engine/engine/content/builtins/render/default.render_script

```diff
@@ -204,9 +203,19 @@ function update(self)
 
     render.enable_state(graphics.STATE_STENCIL_TEST)
     render.draw(predicates.gui, camera_gui.frustum)
-    render.draw(predicates.debug_text, camera_gui.frustum)
     render.disable_state(graphics.STATE_STENCIL_TEST)
     render.disable_state(graphics.STATE_BLEND)
+
+    -- render debug lines and text only in debug
+    --
+    if self.is_debug then
+        render.set_view(camera_world.view)
+        render.set_projection(camera_world.proj)
+        render.enable_state(graphics.STATE_BLEND)
+        render.draw_debug3d() -- debug lines
+        render.draw(predicates.debug_text, camera_gui.frustum) -- debug text
```

Comment:
Using the camera_gui.frustum, while the camera_world view and proj are used above. Is that correct?